data/ecosystems/o/oasis.toml

```diff
@@ -541,3 +541,114 @@ url = "https://github.com/viraja1/privacy_compute_platform"
 
 [[repo]]
 url = "https://github.com/zoek1/FastGoverment"
+
+[[repo]]
+url = "https://github.com/therealbryanho/super-chad-adventures/blob/main/README.md"
```

Comment:
These urls need to be the url of the root gitrepo: https://github.com/therealbryanho/super-chad-adventures